basics/a54.go

```diff
@@ -0,0 +1,35 @@
+// tuple assignment x = y = y, x or multiple variable assignment
+package main
+
+import (
+	"fmt"
+	"math"
+)
+
+func main() {
+	var a, b ,c, d int
+	a = 10
+	b = 5
+	c = 25
+	d = 100
+	fmt.Println("Initial  value of a, b, c, d are ", a, b, c, d)
+	// default way
+	a , b = c , d
+	fmt.Println("After tuple assignment a, b = c,d\nvalue of a, b, c, d are ", a, b, c, d)
+	a = 10
+	b = 5
+	c = 25
+	d = 100
```

Comment:
Values of c and d are unchanged. We don't have to reassign them here.

Reply:
Updated the code